frontend/components/CreateMatchForm.js

```diff
@@ -0,0 +1,215 @@
+import React, {useEffect, useState} from 'react';
+import FormHeader from "./util/FormHeader";
+import {Button, Center, Flex, FormControl, Text, VStack} from "native-base";
+import Api from "./util/Api";
+import {FontAwesome} from "@expo/vector-icons";
+import DateTimePicker from "@react-native-community/datetimepicker"
+import SearchableDropdown from "react-native-searchable-dropdown";
+import {Platform, StyleSheet} from "react-native";
+
+const CreateMatchForm = () => {
+    const [firstPlayer, setFirstPlayer] = useState({})
+    const [secondPlayer, setSecondPlayer] = useState({})
+
+    const [players, setPlayers] = useState([])
+
+    const [date, setDate] = useState(new Date())
+    const [mode , setMode] = useState('date')
+    const [show, setShow] = useState(false)
+    const [showSearch1, setShowSearch1] = useState(true)
+    const [showSearch2, setShowSearch2] = useState(true)
+
+    const [dateString, setDateString] = useState(date.getDate() + "-" + date.getMonth() + "-" + date.getFullYear())
+    const [timeString, setTimeString] = useState(date.getMinutes() < 10 ? date.getHours() + ":0" + date.getMinutes() : date.getHours() + ":" + date.getMinutes())
```

Comment:
Cast to string and padStart function instead of ternary operator.
```suggestion
    const [timeString, setTimeString] = useState(date.getHours() + ":" + date.getMinutes().toString().padStart(2, "0"))
```